crontabber/app.py

```diff
@@ -1235,6 +1246,32 @@ def _configtest_one(self, config):
             )
             return False
 
+    def sentrytest(self):
+        """return true if we managed to send a sample raven exception"""
+        if not (self.config.sentry and self.config.sentry.dsn):
+            raise SentryConfigurationError('sentry dsn not configured')
+
+        try:
+            version = raven.fetch_package_version('crontabber')
+        except Exception:
```

Comment:
I have no idea what kinds of errors that might raise. This is actually, one of those few rare cases where I think it's ok to use `raise Exception`. Because it's a third-party thing.